### DISCUSSION TITLE
Hunter Roam Remote

### DISCUSSION THREAD

**Discussion by Crsarmv7l** [merbanan/rtl_433#2690] [Q&A]
Got a weird one. Attached is a zip of a few captured transmissions from a Hunter Roam Remote. I do have the remote in question and can record more variations, but I am having a heck of a time decoding it.

The actual remote has a few parameters: Address: the address of the receiver needs to match the address of the remote (user changable), Duration: the duration to turn the sprinklers on for, and Program: which program to run.

Addresses span from 0 -127. Programs are 1-24, and duration's can be adjusted from 15min to 45 (need to check that 45 part).

[Hunter.zip](https://github.com/merbanan/rtl_433/files/13126721/Hunter.zip)

The following recordings are in the zip:
Address: 0        Program: 1  Duration: 15
Address: 127   Program: 1  Duration: 15
Address: 0       Program: 2  Duration: 15
Address: 127   Program: 2 Duration: 15

In my attempts to decode, I can see they inserted pauses, and I know this is fixed code.

Appreciate any possible help

**Comment by zuckschwerdt** [comment]
Don't remove meta data from the filenames. Is this 250k at 433.92M?

**Reply by Crsarmv7l** [reply]
Sorry, yes it is.

Someone was able to decode their wired protocol here https://github.com/seb821/OpenSprinkler-Firmware-Hunter/blob/master/hunter.cpp

Which may provide some clues as to the RF signals. Even with that I was still not able to decipher

**Reply by zuckschwerdt** [reply]
The signal is clipping and thus the waveform can't be demodulated well. You need bigger distance or a low fixed gain.
Interesting about the very quiet (3rd) part in the transmissions. Is it bidirectional?

The code indicates normal PWM (208 / 1875 µs) with some preamble.

**Reply by Crsarmv7l** [reply]
I will make some new recordings tonight. I don't believe it is bidirectional.

**Reply by Crsarmv7l** [reply]
That code is for the wired. I am just guessing they wouldn't want to reinvent the wheel with the RF receiver.

**Comment by Crsarmv7l** [comment]
New set. 
[Hunter.zip](https://github.com/merbanan/rtl_433/files/13128228/Hunter.zip)
Used -g -2. They look a lot better. Same signal for all so yes likely the info is encoded int the gaps. Haven't looked closely at them yet. The beginning of each file has A for address and P for program.

**Reply by zuckschwerdt** [reply]
The signal looks good. But it is narrow GFSK which our demod can't quite handle.
![gfsk](https://github.com/merbanan/rtl_433/assets/12833/81a16046-1399-4303-bfa5-48e08b4c3f1a)
If you zoom just right in https://triq.org/spectrogram-next/ then you can seen the coded data
![gfsk2](https://github.com/merbanan/rtl_433/assets/12833/604d4f91-73d9-46ab-add8-bede6ff9b430)

**Reply by Crsarmv7l** [reply]
Thanks. Any other tools you recommend for analyzing it? URH doesn't seem to handle it very well, and I am looking to create a synthetic.

**Comment by Crsarmv7l** [comment]
Hmm after taking a look at these more closely, I don't see it. They all look the same with similar gaps
